src/DeviceInterfaceValues.tsx

```diff
@@ -561,6 +561,16 @@ export default (): React.ReactElement => {
           <BackButton href={`/devices/${deviceId}/edit`} />
           Interface Data
         </h2>
+        {astarte.token?.can(
+          'appEngine',
+          'GET',
+          `/devices/${deviceId}/interfaces/${interfaceName}`,
+        ) &&
+          iface?.ownership === 'device' && (
```

Comment:
Here we should check for datastream interfaces. No matter if its ownership is server or device